### PR TITLE
Fix for newsroom

### PR DESCRIPTION
**Pull request by richaagarwal**
@kave

### CODE REVIEW

**Comment by anselmbradford** [comment]
👍

File: cfgov/v1/models/browse_filterable_page.py [hunk 1]
```diff
@@ -68,18 +68,17 @@ def get_form_class(self):
...
-                    if 'blog' in categories:
-                        del categories[categories.index('blog')]
-                    if not categories:
-                        only_blog = True
-
+                if categories:
+                    if 'blog' not in categories:
+                        get_blog = False
+                    else:
+                        if len(categories) == 1:
```
Comment (sebworks) [inline cfgov/v1/models/browse_filterable_page.py]: can you use  `elif ` here?

**Comment by sebworks** [comment]
Are we changing the logic from only including the blog if no categories to always include the blog?

File: cfgov/v1/models/browse_filterable_page.py (hunk 1 above)
Comment (richaagarwal) [inline cfgov/v1/models/browse_filterable_page.py]: @sebworks how so?

Comment (anselmbradford) [inline cfgov/v1/models/browse_filterable_page.py]: +1 @sebworks ... I think he means:
```
if categories:
    if 'blog' not in categories:
        get_blog = False
    elif len(categories) == 1:
        only_blog = True
```

Comment (richaagarwal) [inline cfgov/v1/models/browse_filterable_page.py]: Oh, right :P  Hmm. I like `elif` for when there are multiple conditions that are unrelated, e.g. `if x == 0: ... elif x == 1: ... elif x == 2: ... ` but as simply a combination of `else` and `if` it seems to be less readable.  Does anyone have stronger thoughts on this?

Comment (sebworks) [inline cfgov/v1/models/browse_filterable_page.py]: ok, that explanation works for me.

**Comment by sebworks** [comment]
👍

File: cfgov/v1/models/browse_filterable_page.py (hunk 1 above)
Comment (sebworks) [inline cfgov/v1/models/browse_filterable_page.py]: I would just add it might help to add comments here. Just so people understand the underlying reasons for the logic.
